redbeams-api/src/test/java/com/sequenceiq/redbeams/validation/DatabaseVendorAndServiceValidatorTest.java

```diff
@@ -23,44 +25,38 @@
 import org.mockito.Mock;
 
 import com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor;
-import com.sequenceiq.cloudbreak.api.endpoint.v4.database.requests.DatabaseV4Request;
-import com.sequenceiq.cloudbreak.validation.externaldatabase.RdsRequestValidator;
+import com.sequenceiq.cloudbreak.util.DatabaseCommon;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.request.DatabaseV4Request;
 
 @RunWith(Parameterized.class)
-public class RdsRequestValidatorTest {
+public class DatabaseVendorAndServiceValidatorTest {
 
-    @Mock
-    private ConstraintValidatorContext constraintValidatorContext;
-
-    @Mock
-    private ConstraintViolationBuilder constraintViolationBuilder;
-
-    @Mock
-    private NodeBuilderCustomizableContext nodeBuilderCustomizableContext;
-
-    private final RdsRequestValidator rdsRequestValidator = new RdsRequestValidator();
+    @Mock(answer = RETURNS_DEEP_STUBS)
```

Comment:
Just a note, again my personal taste: it is another example why I do not like static classes. Were ValidatorUtil an ordinary bean, there would be no need to use DEEP_STUBS.

Reply:
Right. Reworking ValidatorUtil to be an ordinary bean / component should be done, but I don't want to do it for this PR - ValidatorUtil is widely used outside of this validator, so it should be changed separately.